<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementSourceWindow</class>
 <widget class="QWidget" name="MeasurementSourceWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Source</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../resources/icons/buttons/buttonicons.qrc">
    <normaloff>:/icons/buttons/application.ico</normaloff>:/icons/buttons/application.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="maxAndMinMeasurementGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Max and min measurement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="maxMeasurementDoubleSpinBox">
        <property name="toolTip">
         <string/>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-99999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="minMeasurementDoubleSpinBox">
        <property name="toolTip">
         <string/>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-99999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Max measurement:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="measurementNameLineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min measurement:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Measurement name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max raw value:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Min raw value:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="minRawValueDoubleSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-65535.000000000000000</double>
        </property>
        <property name="maximum">
         <double>65535.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="maxRawValueDoubleSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-65535.000000000000000</double>
        </property>
        <property name="maximum">
         <double>65535.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../../resources/icons/buttons/buttonicons.qrc"/>
 </resources>
 <connections/>
</ui>
